apps/frontend/src/app/transformations-creator/transformations-creator.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { FormGroup, FormControl, Validators } from '@angular/forms';
+
+
+interface Transformation {
+  id: number;
+  name: string | null;
+  description: string | null;
+}
+
+@Component({
+  selector: 'app-transformations-creator',
+  templateUrl: './transformations-creator.component.html',
+  styleUrls: ['./transformations-creator.component.scss'],
+})
+
+export class TransformationsCreatorComponent {
+
+  transformations: Transformation[] = [] ;
+
```

Comment:
```suggestion
get nameControl() {
	return this.transformationForm.controls['name']
}
```

This allows you to easily access the control either in typescript or in the template.

Reply:
functioned declared and use it in HTML